src/stories/StyledCoinListItem.tsx

```diff
@@ -22,13 +23,13 @@ const NAME_MAPS: Record<AssetIconVariant, string> = {
   [AssetIconVariant.BTC]: 'Bitcoin',
 };
 
-export default function StyledCoinListItem({ asset, onClick, protocol }: StyledCoinListItemProps) {
+export default function StyledCoinListItem({ asset, onClick, protocol, disabled }: StyledCoinListItemProps) {
+  let buttonClasses = 'flex gap-2 rounded px-3 py-2 h-12';
+
+  disabled ? null : (buttonClasses += ' hover:bg-dfxGray-400/50 focus:bg-dfxGray-400/50 active:bg-dfxGray-400/80');
```

Comment:
```ts
!disabled && (buttonClasses += ' hover:bg-dfxGray-400/50 focus:bg-dfxGray-400/50 active:bg-dfxGray-400/80');
```

Reply:
in the current version the code is already more complex, so the shortform can't be used here imo

`disabled || asset.comingSoon
    ? null
    : (buttonClasses += ' hover:bg-dfxGray-400/50 focus:bg-dfxGray-400/50 active:bg-dfxGray-400/80');`

Reply:
Not 100% sure, but it might still work like this
```ts
(disabled || asset.comingSoon) && (buttonClasses += ' hover:bg-dfxGray-400/50 focus:bg-dfxGray-400/50 active:bg-dfxGray-400/80');
```
or with an if
```ts
if (disabled || asset.comingSoon)
   buttonClasses += ' hover:bg-dfxGray-400/50 focus:bg-dfxGray-400/50 active:bg-dfxGray-400/80';
```

Reply:
some tinkering got me there, had to negate the whole OR-expression. 

`if (!(disabled || asset.comingSoon))
    buttonClasses += ' hover:bg-dfxGray-400/50 focus:bg-dfxGray-400/50 active:bg-dfxGray-400/80';`

Reply:
Ah yes, you're right 😊